pyfolio/plotting.py

```diff
@@ -30,18 +30,32 @@
 from . import pos
 
 
-def set_plot_defaults():
-    """
-    Sets default plotting/charting colors/styles.
-    """
+def plotting_context(func):
+    """Decorator to set plotting context during function call."""
+    def call_w_context(*args, **kwargs):
+        set_context = kwargs.pop('set_context', True)
+        if set_context:
+            with context():
+                return func(*args, **kwargs)
+        else:
+            return func(*args, **kwargs)
+    return call_w_context
+
+
+def context(context='notebook', font_scale=1.5, rc=None):
```

Comment:
The other issue I think would be that we would always override a user-set style (e.g. via matplotlibrc file) as you either pass in your own, or use the default one, but you can't disable.